src/SSVOpenHexagon/Core/MenuGame.cpp

```diff
@@ -1320,30 +1320,34 @@ void MenuGame::drawLevelSelection()
         "(" + toStr(currentIndex + 1) + "/" + toStr(levelDataIds.size()) + ")",
         txtLMus, {20.f, getGlobalTop(lname) - 30.f});
 
-    std::string packNames{"Installed packs:\n"};
+    string packNames{"Installed packs:\n"}, curPack, longestPackName;
```

Comment:
In general, please do not change the coding style (e.g. getting rid of `std::` or of curly braces) -- I will revert these changes prior to merging anyway.